linera-storage/src/lib.rs

```diff
@@ -78,12 +78,29 @@ pub trait Store: Sized {
     /// Writes several values
     async fn write_values(&self, values: &[HashedValue]) -> Result<(), ViewError>;
 
+    /// Writes several values in a batch
+    fn write_values_batch(
```

Comment:
Maybe we should avoid the word "write" for adding things to a batch, if they are not actually being written to storage at that point.

Reply:
yes, exactly!